pandas/core/generic.py

```diff
@@ -7085,6 +7125,28 @@ def interpolate(
                 "column to a numeric dtype."
             )
 
+        # Set `limit_direction` depending on `method`
+        if (method == "pad") or (method == "ffill"):
+            if (limit_direction == "backward") or (limit_direction == "both"):
+                raise ValueError(
+                    "`limit_direction` must not be `%s` for method `%s`"
```

Comment:
You can use f-strings for this and the one on L 7140

Reply:
okay